libraries/classes/InsertEdit.php

```diff
@@ -1802,7 +1802,7 @@ public function getCurrentValueForDifferentTypes(
         } elseif (
             ! ($type === 'datetime' || $type === 'timestamp' || $type === 'date')
             || ($currentValue !== 'CURRENT_TIMESTAMP'
-                && $currentValue !== 'current_timestamp()')
+                && ! preg_match('/^current_timestamp\([0-6]?\)$/', $currentValue))
```

Comment:
This should be a case-insensitive comparison:
```suggestion
            || ! preg_match('/^current_timestamp(\([0-6]?\))?$/i', $value)
```
Same for the other change.